packit/distgit.py

```diff
@@ -455,11 +458,99 @@ def get_latest_build_in_tag(downstream_package_name, dist_git_branch):
 
         return koji_build_str
 
+    @staticmethod
+    def get_changelog_since_latest_stable_build(
```

Comment:
ah oki, understood; `strict=False` makes sense then :)

Reply:
Yes, but I was looking at Koji code and there are all sorts of exceptions that could be raised despite `strict` being set to `False`, and some Koji calls don't even have the strict parameter. So it will be better to do a proper exception handling for each call.